src/output/log.ts

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-console */
+
+import chalk from 'chalk';
+
+const libraryName = '[next-runtime-env]';
```

Comment:
I initially included this prefix, but now that we follow Next.js's style, wouldn't it look cleaner without it? (Almost like a native integration)

Reply:
Yes, it might be cleaner. I do think it's nice for developers to know what package is creating these lines in case something goes wrong or they need to debug it.

Reply:
Agreed. Looking at the screenshot above, it looks nice. Let's keep it.